lib/pure/json.nim

```diff
@@ -656,6 +656,8 @@ proc escapeJson*(s: string): string =
   result = newStringOfCap(s.len + s.len shr 3)
   escapeJson(s, result)
 
+proc toUgly*(result: var string, node: JsonNode)
```

Comment:
This breaks Nim's effect inference.

Reply:
I can reorder the procedures, no problem, but that makes the diff larger.